docs/src/pages/guides/routing/routing.md

```diff
@@ -58,23 +58,34 @@ const theme = createTheme({
 ## `component` prop
 
 You can achieve the integration with third-party routing libraries with the `component` prop.
-You can learn more about this prop in the [composition guide](/guides/composition/#component-prop).
+You can learn more about this prop in the [**composition guide**](/guides/composition/#component-prop).
+
+### Link
 
 Here are a few demos with [react-router](https://github.com/ReactTraining/react-router).
 You can apply the same strategy with all the components: BottomNavigation, Card, etc.
 
-### Link
-
 {{"demo": "pages/guides/routing/LinkRouter.js"}}
 
-### Tabs
-
-{{"demo": "pages/guides/routing/TabsRouter.js", "defaultCodeOpen": false}}
-
 ### Button
 
 {{"demo": "pages/guides/routing/ButtonRouter.js"}}
 
+**Note**: The button base component adds the `role="button"` attribute when it identifies the intent to render a button without a native `<button>` element.
+This can create issues when rendering a link.
+If you are not using one of the `href`, `to`, or `component="a"` props, you need to **swallow the role** attribute.
+It's why `role={undefined}` is set in the above demo.
```

Comment:
```suggestion
If you are not using one of the `href`, `to`, or `component="a"` props, you need to override the `role` attribute.
The above demo achieves this by setting `role={undefined}` **after** the spread props.
```

Reply:
@oliviertassinari should I do these changes?